etc/httpd/conf.d/status.conf

```diff
@@ -0,0 +1,10 @@
+ExtendedStatus on
+<VirtualHost 127.0.0.1:80>
+    ServerName local-stackdriver-agent.stackdriver.com
+    <Location /server-status>
+        SetHandler server-status
+        Order deny,allow
+        Deny from all
+        Allow from 127.0.0.1
+    </Location>
+</VirtualHost>
```

Comment:
Apache on debian seems to be pre-configured with mod_status enabled on `/server-status`. So this is likely to only apply to Red Hat systems. See also the corresponding documentation change.